psga/sars_cov_2/psga.nf

```diff
@@ -5,13 +5,12 @@ if ( params.sequencing_technology == "illumina" ) {
     include { contamination_removal_illumina as contamination_removal } from './common/contamination_removal.nf'
     include { ncov2019_artic_nf_pipeline_illumina as ncov2019_artic } from './ncov2019_artic.nf'
     include { stage_sample_file } from './common/fetch_sample_files.nf'
-    include { stage_sample_file_pair as stage_fastq_sample_file } from './common/fetch_sample_files.nf'
+    include { stage_sample_file_pair } from './common/fetch_sample_files.nf'
     include { bam_to_fastq_illumina as bam_to_fastq } from './common/utils.nf'
 } else if ( params.sequencing_technology == "ont" ) {
     include { contamination_removal_ont as contamination_removal } from './common/contamination_removal.nf'
     include { ncov2019_artic_nf_pipeline_medaka as ncov2019_artic } from './ncov2019_artic.nf'
     include { stage_sample_file } from './common/fetch_sample_files.nf'
-    include { stage_sample_file as stage_fastq_sample_file } from './common/fetch_sample_files.nf'
```

Comment:
The latest nextflow version raises a warning because it dislikes the fact that stage_fastq_sample_file exists twice. We don't really need this, so I cleaned the code to prevent this warning.